backend/clubs/views.py

```diff
@@ -5354,6 +5354,56 @@ def transfer(self, request, *args, **kwargs):
     def get_queryset(self):
         return Ticket.objects.filter(owner=self.request.user.id)
 
+    def give_tickets(self, order_info, cart, reconciliation_id):
```

Comment:
nit: would be nice to make it clear that this is for internal use only.

Reply:
You mean only to be used in `initiate_checkout` and `complete_checkout`?

Reply:
More in the spirit of `_calculate_cart_total`. [Quick read](https://realpython.com/python-double-underscore/#single-leading-underscore-in-python-names).

Reply:
[Less quick read, but still useful](https://stackoverflow.com/questions/1301346/what-is-the-meaning-of-single-and-double-underscore-before-an-object-name)